packages/design-system/src/components/Dropdown/__snapshots__/Select.test.tsx.snap

```diff
@@ -16,12 +17,10 @@ exports[`Dropdown dropdown matches snapshot 1`] = `
     </span>
   </label>
   <select
```

Comment:
Just noticed that the `Select` component is still hanging around. It can be removed now, right?

Reply:
> Yes, unless we need to deprecate it?

Because it's supposed to be internal, I think we could just skip to removal. It's not on our doc site.

Reply:
Yes! 🎉